database/manual-updates/2022-03-17_1-add-start-end-fields-to-workerteam-table-in-postgres.md

```diff
@@ -0,0 +1,27 @@
+# Add start and end fields to the worker team table
+
+## The problem we're trying to solve
+
+We need to add start and end columns to the worker team table in order to be able to run correct reports.
+
+## Justification for doing a manual update
+
+We don't have automated db migrations
+
+## The plan
+
+1. Run the SQL queries below one by one
```

Comment:
minor comment that doesn't affect functionality: as we are not querying the DB but running a command to alter the schema, _queries_ is not the most appropriate word, _script_ might be a better choice here